docs/Customs Declare Exports AutoComplete.js

```diff
@@ -1,7 +1,7 @@
 // ==UserScript==
 // @name         Customs Declare Exports AutoComplete
 // @namespace    http://tampermonkey.net/
-// @version      1.9
+// @version      2.0
```

Comment:
I think it'll be better to stay with `1.10`. We changed this with `major` change last time, but there is no point to upgrade `major` version. Of course it's not a mistake

Reply:
Done